source/stylesheets/card.scss

```diff
@@ -6,7 +6,30 @@
   border-radius: $base-border-radius;
   height: $card-height;
   margin: $base-vh-spacing auto 0;
+  transition: $base-transition;
+  position: absolute;
+  left: 50%;
+  margin-left: -$card-width / 1.77;
+  top: 7.75vh;
+  z-index: -100;
```

Comment:
Is there anything special about this value, or is it arbitrary? It's nice to move these `z-index` values into variables, so we can compare them easily against each other, and see if we're stepping on toes anywhere